lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -39,7 +57,8 @@ define([
 
         describe('Navigation', function () {
             it('does not render breadcrumbs for the top level tabs', function() {
-                var teamsTabView = createTeamsTabView();
+                var requests = AjaxHelpers.requests(this),
```

Comment:
I know, but it felt strange to call the method and drop the object on the floor. The problem is that rendering the view actually triggers events, so we do need Sinon to be in place. I decided it was cleaner to leave the unused variables, but if you disagree I can do something different.